dll/shellext/zipfldr/lang/ro-RO.rc

```diff
@@ -48,15 +49,15 @@ END
 
 IDD_PASSWORD DIALOGEX 0, 0, 243, 62
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Password needed"
+CAPTION "Parolă necesară"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     EDITTEXT        IDC_PASSWORD,72,42,108,13,ES_PASSWORD | ES_AUTOHSCROLL
-    DEFPUSHBUTTON   "OK",IDOK,186,6,50,14
-    PUSHBUTTON      "Skip File",IDSKIP,186,24,50,14
-    PUSHBUTTON      "Cancel",IDCANCEL,186,42,50,14
+    DEFPUSHBUTTON   "Confirmă",IDOK,186,24,57,14
+    PUSHBUTTON      "Sari peste fișier",IDSKIP,186,24,57,14
```

Comment:
**Critical:** Two buttons are placed in one position (probably by accident, will fix that).